mgmtd/mgmt_txn.c

```diff
@@ -110,7 +110,6 @@ struct mgmt_commit_cfg_req {
 
 	/* Track commit phases */
 	enum mgmt_commit_phase curr_phase;
```

Comment:
It might be reasonable to change the naming of this variable too? Just to be `phase` in we don't have current/next semantics?

Reply:
Done.